example/src/main/java/com/stripe/example/activity/IDEALPaymentMethodActivity.kt

```diff
@@ -0,0 +1,154 @@
+package com.stripe.example.activity
+
+import android.os.Bundle
+import androidx.activity.compose.setContent
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material.Button
+import androidx.compose.material.DropdownMenu
+import androidx.compose.material.DropdownMenuItem
+import androidx.compose.material.Icon
+import androidx.compose.material.LinearProgressIndicator
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.OutlinedTextField
+import androidx.compose.material.Text
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.ArrowDropDown
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.livedata.observeAsState
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import androidx.lifecycle.lifecycleScope
+import com.stripe.android.model.Address
+import com.stripe.android.model.PaymentMethod
+import com.stripe.android.model.PaymentMethodCreateParams
+
+class IDEALPaymentMethodActivity : StripeIntentActivity() {
+    private val bankNameMap = mapOf(
+        "abn_amro" to "ABN AMRO",
+        "asn_bank" to "ASN Bank",
+        "bunq" to "Bunq",
+        "handelsbanken" to "Handelsbanken",
+        "ing" to "ING",
+        "knab" to "Knab",
+        "moneyou" to "Moneyou",
+        "rabobank" to "Rabobank",
+        "revolut" to "Revolut",
+        "regiobank" to "RegioBank",
+        "sns_bank" to "SNS Bank (De Volksbank)",
+        "triodos_bank" to "Triodos Bank",
+        "van_lanschot" to "Van Lanschot"
+    )
```

Comment:
I think this should be available from the SDK to users, we don't expose that?

Reply:
It is available in paymentsheet dropdown config. I can look into extracting this out.

Reply:
I looked into this more, this is currently coming from `lpms.json`, but will be coming from the server. I think that if the users want this list, they can create it manually on their server side, wdyt? We can keep this example the way it is for now.